clustermesh-apiserver/clustermesh/root.go

```diff
@@ -60,10 +60,11 @@ func NewCmd(h *hive.Hive) *cobra.Command {
 		PreRun: func(cmd *cobra.Command, args []string) {
 			// Overwrite the metrics namespace with the one specific for the ClusterMesh API Server
 			metrics.Namespace = metrics.CiliumClusterMeshAPIServerNamespace
-			option.Config.Populate(h.Viper())
+			option.Config.PopulateLogging(h.Viper())
 			if option.Config.Debug {
```

Comment:
Could this be part of the PopulateLogging call?

Reply:
There are too many callers calling this Populate or now PopulateLogging, and their logging setup are a bit different. I'd rather leave them to the individual callers.

Reply:
Hmm, I'd be personally in favor of moving the logic to PopulateLogging as well, to ensure uniformity across all components. The only part which seems different is the tag, which could be easily provided as a parameter. 

As for the clustermesh component, the additional part would be also adding the `LogOpt` and `LogDriver` parameters to `DefaultLegacyClusterMeshConfig`, similarly to https://github.com/cilium/cilium/commit/1f29e774912cd932e4ebca93145419a71e631366.

Reply:
What about the unit test cases where we populate but don't actually setup logging?

Reply:
Dropping the PopulateLogging call altogether from unit tests should be fine, assuming that the variables are never used afterwards.

Reply:
Okay let's choose a different name then. It's not just "Populate" anymore. What about `option.Config.SetupLogging()`?